EDASH-63: Dependency Updating and Long Needed Maintenance

martin-bucinskas opened the pull request:
## Description
Bumped up some of the dependency versions from Stone Age to modern world.
Now using react-scripts@3.1.1 which makes the application more react like.
No longer using long paths that may break when moving things around in components.

## JIRA Ticket (if applicable)
https://transformuk.atlassian.net/browse/EDASH-63

Commit:
EDASH-63: Dependency version bump

Non breaking change dependency version bump for axios and googleapis.

Commit:
EDASH-63: Updated dependencies.

This is now using the modern day dependencies, let the stone age ones rip.
Some black magic with package-lock to fix the breaking changes for react-scripts.

chriscoffee commented:
LGTM

Could you convert (with imagemagick or something) all the images from jpg -> png?

Also run them through tinypng or similar so we can make sure it's small as fuck.

Commit:
EDASH-63: Image Optimisation.

 - Converted images to .png from .jpg with imagemagick.
 - Used tinypng to reduce size of images.

Commit:
EDASH63: Added images and css.

For some reason the images did not get added.

Commit:
EDASH-63: Removed the gallery jpg.

Gallery now should use .png images.

Commit:
EDASH-63: Correctly resized images.

Did not resize the previous images. Now they are.

martin-bucinskas committed:
Merge branch 'master' into EDASH-63
